code/modules/mob/living/carbon/human/species_types/pumpkin_man.dm

```diff
@@ -1,4 +1,4 @@
-/datum/species/pod/pumpkin_man
+/datum/species/pumpkin_man
```

Comment:
Brought this up in discord - make sure this race has everything defined it needs to in order to be a standalone race since podpeople were removed.